package.json

```diff
@@ -47,11 +47,11 @@
   "dependencies": {
     "ember-cli-babel": "^7.26.11",
     "ember-cli-htmlbars": "^6.1.1",
-    "ember-cli-typescript": "^5.2.1",
-    "ember-export-application-global": "^2.0.1"
+    "ember-cli-typescript": "^5.2.1"
   },
   "devDependencies": {
     "@ember/optional-features": "^2.0.0",
+    "@ember/string": "^3.1.1",
```

Comment:
Should also be a `peerDependencies`

(in the past `@ember/string` was provided by `ember-data` via `dependencies`, but in recent versions `ember-data` moved it to `peerDependencies`, so as we consume it in `/addon` we have either to define it in `dependencies` or `peerDependencies`, as `ember-data` put it in `peerDependencies` as guess we can stick with having it in `peerDependencies`, see https://github.com/emberjs/data/blob/main/packages/-ember-data/package.json#L19-L21 ; https://github.com/emberjs/data/pull/8423 ; https://github.com/emberjs/data/issues/8393)

Reply:
updated